crates/sui-framework/packages/dwallet-system/sources/sui_state_proof.move

```diff
@@ -164,9 +176,8 @@ module dwallet_system::sui_state_proof {
         
         let (cap_ids_serialised_bytes, messages_serialised_bytes) = sui_state_proof_verify_transaction(committee.committee, checkpoint_summary, checkpoint_contents, transaction, config.approve_event_type_layout, config.package_id );
 
-
-        let messages = bcs::peel_vec_vec_u8(&mut bcs::new(messages_serialised_bytes));
         let cap_ids = bcs::peel_vec_address(&mut bcs::new(cap_ids_serialised_bytes));
```

Comment:
technically it could be that in the same txn several dwallet caps on sui requested signing of a message. then it would be wrong to just return a single one. would keep it like this.